src/services/trucService.js

```diff
@@ -0,0 +1,50 @@
+import firebase from "../utils/firebaseConfig";
+
+const trucService = {
+  findAll: (trucDBName) => {
```

Comment:
having a dynamic db name is pretty hard to maintain in my opinion. It forces us to keep the exact same schema for all our databases which is not really cool. Can we stick to one db per service ? 🙏